src/telemetry/events.ts

```diff
@@ -36,3 +36,8 @@ export const TM_EVENT_PROJECT_CLOUD = "execute.project.cloud";
 // events for language server
 export const TM_EVENT_LANG_SERVER = "ballerina.langserver.event";
 export const TM_ERROR_LANG_SERVER = "ballerina.langserver.error";
+export const TM_FEATURE_USAGE_LANG_SERVER = "ballerina.langserver.feature.usage";
+
+// Language server telemetry event types
+export const TM_EVENT_TYPE_ERROR = "ErrorTelemetryEvent";
```

Comment:
Shall we move these constants to activator.ts as these aren't telemetry event names that we send?

Reply:
Done! Moved.